### ISSUE TITLE
Getting more done in GitHub with ZenHub

### ISSUE THREAD

**Issue by Techievena** [coala/coala#4117]
Hola! @Techievena has created a [ZenHub](http://www.zenhub.com) account for the **coala** organization. ZenHub is the only project management tool integrated natively in GitHub – created specifically for fast-moving, software-driven teams.

----

#### How do I use ZenHub?

To get set up with ZenHub, all you have to do is **[download the browser extension](https://www.zenhub.com?utm_source=ZHOnboarding)** and log in with your GitHub account. Once you do, you’ll get access to ZenHub’s complete feature-set immediately.

#### What can ZenHub do?

ZenHub adds a series of enhancements directly inside the GitHub UI:

- Real-time, customizable task boards for GitHub issues;
- Multi-Repository burndown charts, estimates, and velocity tracking based on GitHub Milestones;
- Personal to-do lists and task prioritization;
- Time-saving shortcuts – like a quick repo switcher, a “Move issue” button, and much more.

### [Add ZenHub to GitHub](https://www.zenhub.com?utm_source=ZHOnboarding)

_Still curious? See [more ZenHub features](https://www.zenhub.com/features?utm_source=ZHOnboarding) or read [user reviews](https://chrome.google.com/webstore/detail/zenhub-for-github/ogcgkffhplmphkaahpmffcafajaocjbd/reviews). This issue was written by your friendly ZenHub bot, posted by request from @Techievena._

![ZenHub Board](https://cloud.githubusercontent.com/assets/8771909/11153956/233ac4a8-89f1-11e5-94b1-1569d3f38b4d.png)

**Comment by Techievena** [comment]
Can we use this integration? Looks awesome

**Comment by jayvdb** [comment]
The biggest problem is the permission system is based on GitHub.
https://github.com/ZenHubIO/support/issues/910

Also, I found performance/responsiveness on Firefox to be a problem. https://github.com/ZenHubIO/support/issues/1032

But the most annoying problem is that once you install ZenHub , it wants to take over all repos, which is both a usability problem and an inappropriate business model.
(https://github.com/ZenHubIO/support/issues/722  / https://github.com/ZenHubIO/support/issues/757 )

Then there is the problem that the data is not visible to ordinary GitHub users who dont use ZenHub ( https://github.com/ZenHubIO/support/issues/754 ), and it is a proprietary SaaS and its longevity is not assured so we'd want to ensure that we can export our data (https://github.com/ZenHubIO/support/issues/1070).

So, IMO, no.

**Comment by Makman2** [comment]
looks like https://waffle.io/

**Comment by Techievena** [comment]
Can we add these features to Gitmate?

**Comment by Nosferatul** [comment]
also, we have `projects` from github

**Comment by adtac** [comment]
To all maintainers: be careful logging in with coala authorization. I did that by mistake the last time with waffle and we got a bunch of unnecessary labels (redundant) added to our repo's label list.

**Comment by jayvdb** [comment]
IMO the org should disable integrations being added automatically.

**Comment by adtac** [comment]
@jayvdb I'm looking at https://github.com/organizations/coala/settings/oauth_application_policy - is that what we want?

**Comment by meetmangukiya** [comment]
Yes, is it disabled? While trying other apps you cannot disable coala for that :P
